bin/improver-wind-direction

```diff
@@ -42,7 +42,18 @@ def main():
     """Load in arguments to calculate mean wind direction from ensemble
        realizations."""
 
+    cli_specific_arguments = [(['--backup_method'],
+                               {'dest': 'backup_method',
+                                'default': 'neighbourhood',
+                                'help': ('Backup method to use if '
+                                         'there is low confidence in'
+                                         ' the wind_direction. '
+                                         'options are first_realization'
+                                         ' or neighbourhood. '
+                                         'Default is neighbourhood.')})]
```

Comment:
As discussed we'd rather do this explicitly at the CLI level than catching the "non-equal area grid" error in the plugin.  So could we specify in this help string please that the "first_realization" option is being provided for global data.  (This can be updated later if need be after the verification trials for the two methods.)

Reply:
Done